### PR TITLE
Hexagon improvements

### PR DESCRIPTION
**Pull request by tom-sherman**
- Disable animation for those that prefer reduced motion
- Make the bestagons twinkle ✨

### CODE REVIEW

**Comment by tom-sherman** [comment]
These twinkles are janky af lol, way too much time spent to figure out which ones should animate.

File: components/Hero.jsx [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+import { useEffect, useState } from "react";
+
+const hexArray = Array.from({length: 8 * 12}).fill(null);
+
 export default function Hero(/* { speakers } */) {
+  const [highlightedHexes, setHighlightedHexes] = useState([]);
+
+  useEffect(() => {
+    const timeout = setInterval(() => {
+      console.log("fkdhjdkfh")
```
Comment (damiensedgwick) [inline components/Hero.jsx:11]: lol what is this...

Comment (tom-sherman) [reply components/Hero.jsx]: shhh...

**Comment by tom-sherman** [comment]
Closing in favour of #70

**Pull request closed without merging**